segmentstore/storage/src/main/java/io/pravega/segmentstore/storage/chunklayer/ChunkInfo.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.segmentstore.storage.chunklayer;
+
+import lombok.Builder;
+import lombok.Data;
+import lombok.NonNull;
+
+/**
+ * Chunk Information.
+ */
+@Builder
+@Data
+final public class ChunkInfo {
```

Comment:
What is the purpose of this class ?
Given a chunkname you can get its current length

is it expected to be used by tools or only internally ?

Reply:
The purpose of this class is to get information related to chunk from ChunkStorageProvider. 
The only relevant property today is length, but in future their could be more.